dvc/command/pipeline.py

```diff
@@ -236,7 +236,7 @@ def add_parser(subparsers, parent_parser):
     )
     pipeline_show_parser.set_defaults(func=CmdPipelineShow)
 
-    PIPELINE_LIST_HELP = "List pipelines."
+    PIPELINE_LIST_HELP = "List connected groups of stages (pipelines)."
```

Comment:
Why are we making this clarification here, but not in other places? `pipeline` is self descriptive enough, I don't think it needs a "connected groups of stages"

Reply:
Good question. Again, this is to match docs, https://dvc.org/doc/command-reference/pipeline/list specifically.

> but not in other places

The only other place really is pipeline show which reads "Show stages in a pipeline." This one also mentions both terms "stage" and "pipeline" so it's similarly clear, I think.